shortcuts.js

```diff
@@ -0,0 +1,391 @@
+const AppConstants = require("resource://gre/modules/AppConstants.jsm");
```

Comment:
I for some reason aren't getting notifications. This will require a complete recode so I'm going to put this old hold for the time being.